underlay/src/main/java/bio/terra/tanagra/underlay/entitymodel/entitygroup/CriteriaOccurrence.java

```diff
@@ -102,15 +112,32 @@ public Relationship getPrimaryCriteriaRelationship() {
 
   public boolean hasInstanceLevelDisplayHints(Entity occurrenceEntity) {
     return !occurrenceAttributesWithInstanceLevelDisplayHints
+            .get(occurrenceEntity.getName())
+            .isEmpty()
+        || !occurrenceAttributesWithRollupInstanceLevelDisplayHints
+            .get(occurrenceEntity.getName())
+            .isEmpty();
+  }
+
+  public boolean hasRollupInstanceLevelDisplayHints(Entity occurrenceEntity) {
+    return !occurrenceAttributesWithRollupInstanceLevelDisplayHints
         .get(occurrenceEntity.getName())
         .isEmpty();
   }
 
-  public ImmutableSet<Attribute> getAttributesWithInstanceLevelDisplayHints(
+  public ImmutableMap<Attribute, Boolean> getAttributesWithInstanceLevelDisplayHints(
       Entity occurrenceEntity) {
-    return ImmutableSet.copyOf(
-        occurrenceAttributesWithInstanceLevelDisplayHints.get(occurrenceEntity.getName()).stream()
-            .map(occurrenceEntity::getAttribute)
-            .collect(Collectors.toSet()));
+    Map<Attribute, Boolean> merged =
```

Comment:
Do we even need the attributes w/o roll up hints? 
reason for ask: rollupOccCriIdPairKVs = null otherwise